Sources/Passage/Passage.docc/Guide_SessionManagement.md

```diff
@@ -0,0 +1,71 @@
+# Session Management
+
+This guide provides an overview of how session management is handled using Passage and how to use passage-ios to handle refresh tokens.
+
+## Overview
```

Comment:
@flanagankp What do you think about removing everything from Overview to Configuring refresh tokens and linking to the docs? I'd prefer if we only had to make updates in a single location for non-language/framework specific details.

Reply:
Sure I'm on board with that change